shopinvader_api_payment_cart/routers/payment.py

```diff
@@ -0,0 +1,21 @@
+# Copyright Odoo SA (https://odoo.com)
+# Copyright 2024 ACSONE SA (https://acsone.eu).
+# @author Stéphane Bidoul <stephane.bidoul@acsone.eu>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+from odoo.fields import Command
+
+from odoo.addons.shopinvader_api_payment.schemas import TransactionCreate
+
+
+class ShopinvaderApiPaymentRouterHelper(models.AbstractModel):
+    _inherit = "shopinvader_api_payment.payment_router.helper"
+
+    def _get_payable_create_values(self, data: TransactionCreate) -> dict:
+        # Intended to be extended for invoices, carts...
+        payable_create_values = super()._get_payable_create_values(data)
+        payable_model, payable_id = data.payable.split(",")
+        if payable_model == "sale.order":
+            payable_create_values["sale_order_ids"] = [Command.set([int(payable_id)])]
+        return payable_create_values
```

Comment:
This file needs to be in the `models` directory.

Reply:
I was wondering the same. When defining a router helper model, we define it in the router file, under the `routers` directory. But then its extensions go in the `models` directory? 
Same question for other Shopinvader PR's then, such as here https://github.com/shopinvader/odoo-shopinvader/blob/81b5e36a6ca4a3b5117680f2f6392da4c8c2295d/shopinvader_api_sale_loyalty/routers/cart.py#L64